docs/en/stack/ml/anomaly-detection/influencers.asciidoc

```diff
@@ -0,0 +1,77 @@
+[role="xpack"]
+[[ml-influencers]]
+=== Influencers
+
+When anomalous events occur, we want to know why. To determine the cause,
+however, you often need a broader knowledge of the domain. If you have
+suspicions about which entities in your dataset are likely causing
+irregularities, you can identify them as influencers in your {anomaly-jobs}.
+That is to say, _influencers_ are fields that you suspect contain information
+about someone or something that influences or contributes to anomalies in your
+data.
+
+Influencers can be any field in your data. If you use {dfeeds}, however, the
+field must exist in your {dfeed} query or aggregation; otherwise it is not
+included in the job analysis.
+////
+In the case of the {dfeed} query, influencers must exist in the results of the
+query in the same hit as the detector data.
+TBD: Why? 
```

Comment:
This is because when the {dfeed} processes the data, it does so by paging through the results. We try to use `doc_values` when possible, but those `doc_values` are tied to each search hit. Since search hits cannot themselves span multiple indices/documents then the {dfeed} cannot. 

It is possible that influencers and detector data are from different docs/indices when using aggs, both indices though must have the same "time_stamp" field configured.

Reply:
> ... both indices though must have the same "time_stamp" field configured.

Do you mean that the field has to have the same name?  Or format?

Reply:
same name. Aggregations work by aggregating on field names, and that field name must have the same type (date in this case).

Reply:
Thanks, @benwtrent I've added that information in the following sentence: "However, both indices must have a date field with the same name, which you
specify in the `data_description`.`time_field` property for the {dfeed}." If that needs more tweaking, please just let me know!